OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -493,6 +493,9 @@ static bool Map(S server, Connection conn, Session.Client client, string s)
 						if (server.Map.DefinesUnsafeCustomRules)
 							server.SendMessage("This map contains custom rules. Game experience may change.");
 
+						if (server.Map.PlayerCount > 64)
+							server.SendMessage($"The game can't be started because this map exceeds the maximum player limit ({server.Map.PlayerCount}/64).");
```

Comment:
Can we hook this in with `Map.InvalidCustomRules` / `MapPreview.InvalidCustomRules` instead, and remove this?

This is a syntax error with the map, so should be treated the same as other errors. I don't think we should have special case messages like this to players.

Reply:
Dropped the message.

Is the way I hooked the check in with `Map.InvalidCustomRules` fine with you?
```c#
	bool invalidCostumRules;
	public bool InvalidCustomRules { get { return invalidCostumRules || PlayerCount > 64; } }
```

Reply:
I haven't tested it yet, but this looks like it should work aside from the typo.